SwrveSDK/src/main/java/com/swrve/sdk/SwrveBase.java

```diff
@@ -317,6 +320,41 @@ protected void _userUpdate(Map<String, String> attributes) {
         queueEvent("user", parameters, null);
     }
 
+    protected void _userUpdateDate (Map<String, Date> attributes) {
```

Comment:
I was thinking if we package the end result into a map of attributes. might as well have the option to iterate it. Doesn't have to be public facing I think it'll be useful if we want to batch these down the line.